lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/contententry/detailattempttab/ContentEntryDeatilAttemptsSessionListScreen.kt

```diff
@@ -0,0 +1,169 @@
+                            attemptsSessionListItems?.isSuccessful != null -> {
+                                if (attemptsSessionListItems.isSuccessful == true) Icons.Filled.Check
+                                else Icons.Filled.Close
+                            }
+
+                            attemptsSessionListItems?.isCompleted == true -> Icons.Filled.Check
+                            else -> Icons.Filled.Close
+
+                        }, contentDescription = when {
+                            attemptsSessionListItems?.isSuccessful != null -> {
+                                if (attemptsSessionListItems.isSuccessful == true) "Success"
+                                else "Failure"
+                            }
+
+                            attemptsSessionListItems?.isCompleted == true -> "Completed"
+                            else -> "Incomplete"
+                        }, modifier = Modifier.padding(end = 8.dp))
+                },
+                headlineContent = {
+                    androidx.compose.material3.Text(
+                        text = when {
+                            attemptsSessionListItems?.isSuccessful != null -> {
+                                when (attemptsSessionListItems.isSuccessful) {
+                                    true -> "Passed"
+                                    false -> "Failed"
+                                    else -> ""
+                                }
+                            }
+
+                            else -> {
+                                if (attemptsSessionListItems?.isCompleted == true) {
+                                    "Completed"
+                                } else {
+                                    "Incomplete"
+                                }
+                            }
+                        }
+
+                    )
+                },
+                supportingContent = {
+                    Column {
+                        Row(
+                            modifier = Modifier.fillMaxWidth(),
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            Icon(
+                                imageVector = Icons.Filled.Timer,
+                                contentDescription = "Icon",
+                                modifier = Modifier.padding(8.dp)
+                            )
+                            Text(
+                                text =
+                                        "$formattedDateAndTime"
+
+                            )
+
+                        }
+                        Row(
+                            modifier = Modifier.fillMaxWidth(),
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            Icon(
+                                imageVector = Icons.Filled.Star,
+                                contentDescription = "Icon",
```

Comment:
If contentDescription is anything other than blank or null, it must use localized strings the same as any other text used in the interface.

When an icon is used for decoration only - the contentDescription should be null or blank (so that a screen reader for someone with visual impairments will skip it). If the icon has a function (e.g. a button which has only an icon, no text, like a delete button), then the contentDescription should be set.

Reply:
Made the changes accordingly